src/neo/SmartContract/Native/Tokens/Nep17Token.cs

```diff
@@ -22,11 +22,11 @@ public abstract class Nep5Token<TState> : NativeContract
         protected const byte Prefix_TotalSupply = 11;
         protected const byte Prefix_Account = 20;
 
-        protected Nep5Token()
+        protected Nep17Token()
         {
             this.Factor = BigInteger.Pow(10, Decimals);
 
-            Manifest.SupportedStandards = new[] { "NEP-5" };
+            Manifest.SupportedStandards = new[] { "NEP-17" };
```

Comment:
Set `Manifest.Name = Name` ?

Reply:
It was done here https://github.com/neo-project/neo/pull/2024/files#diff-9e0ea43642f8e907463bf6fdc2010e0d87553b4024abc309dfed6501ad407b30R76